File: app/src/main/java/com/babestudios/companieshouse/data/model/filinghistory/DescriptionValues.java
package com.babestudios.companieshouse.data.model.filinghistory;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class DescriptionValues {
	@SerializedName("made_up_date")
	public String madeUpDate;

	@SerializedName("officer_name")
	public String officerName;

	@SerializedName("appointment_date")
	public String appointmentDate;

	@SerializedName("termination_date")
	public String terminationDate;

	@SerializedName("new_date")
	public String newDate;

	@SerializedName("change_date")
	public String changeDate;

	@SerializedName("old_address")
	public String oldAddress;

	@SerializedName("new_address")
	public String newAddress;

	@SerializedName("form_attached")
	public String formAttached;

	@SerializedName("charge_number")
	public String chargeNumber;

	@SerializedName("charge_creation_date")
	public String chargeCreationDate;

	@SerializedName("date")
	public String date;

	@SerializedName("capital")
	public List<Capital> capital = new ArrayList<>();

}

File: data/src/main/java/com/babestudios/companyinfo/data/model/filinghistory/DescriptionValues.kt
package com.babestudios.companyinfo.data.model.filinghistory

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

import java.util.ArrayList

@Parcelize
class DescriptionValues(
		@SerializedName("made_up_date")
		var madeUpDate: String? = null,

		@SerializedName("officer_name")
		var officerName: String? = null,

		@SerializedName("appointment_date")
		var appointmentDate: String? = null,

		@SerializedName("termination_date")
		var terminationDate: String? = null,

		@SerializedName("new_date")
		var newDate: String? = null,

		@SerializedName("change_date")
		var changeDate: String? = null,

		@SerializedName("old_address")
		var oldAddress: String? = null,

		@SerializedName("new_address")
		var newAddress: String? = null,

		@SerializedName("form_attached")
		var formAttached: String? = null,

		@SerializedName("charge_number")
		var chargeNumber: String? = null,

		@SerializedName("charge_creation_date")
		var chargeCreationDate: String? = null,

		@SerializedName("date")
		var date: String? = null,

		@SerializedName("capital")
		var capital: List<Capital> = ArrayList(),

		var description: String? = null
) : Parcelable
